src/arcaflow_plugin_sdk/schema.py

```diff
@@ -5443,7 +5446,7 @@ def _resolve(
             path: typing.Tuple[str],
             scope: ScopeType,
     ) -> typing.Union[AbstractType, PropertyType]:
-        if t == ANY_TYPE:
+        if isinstance(t, typing._SpecialForm):
```

Comment:
Is `_SpecialForm` part of the `typing` interface that is publicly documented and supported?